### DISCUSSION TITLE
Change Docker Internal Port

### DISCUSSION THREAD

**Discussion by Sridou** [rogerfar/rdt-client#665] [General]
**What version are you using?**
v2.0.92
**What OS are you running?**
Linux
**Are you using Docker or as a service?**
Docker
**Which debrid provider are you using?**
Real-Debrid
**Which downloader are you using?**
Internal Downloader

Issue:
How do I change the DOCKER INTERNAL port of 6500 to some other port in Docker Compose file or via settings? As per this [issue](https://github.com/rogerfar/rdt-client/issues/471#issue-2249251284) , I can see that appsettings.json is created when installing rdt-client as a service in linux (I tested), in which we can change the "PORT" setting. I checked my config data path but appsettings.json file has NOT been created. I created it and changed to different port but it's not changing. 

Expected:
To change INTERNAL port to some other port

Use Cases:
Multiple instances of rdt-client
To avoid conflict when running behind gluetun

**Comment by rogerfar** [comment]
If you're using Docker, then just map the port to something else:

```
docker run -d \
  --name=rdtclient \
  -e PUID=1000 \
  -e PGID=1000 \
  -e TZ=Europe/London \
  **-p 6501:6500 \**
  -v <path to data>:/data/db \
  -v <path/to/downloads>:/data/downloads \
  --restart unless-stopped \
  rogerfar/rdtclient
```

**Reply by BrettBedarf** [reply]
Found this after commenting on https://github.com/rogerfar/rdt-client/issues/540#issuecomment-2661979585 but this still conflicts between multiple instances with gluetun case (per OP) since they are both bound to the same localhost. Also since port mappings go in the gluetun service it would end up looking something like this which maybe might make it more clear why this won't work:

```
service: gluetun
ports:
	- 6500:6500 #instance 1
	- 6501:6500 #instance 2... 
```
